frontend/src/components/UploadPipelineDialog.tsx

```diff
@@ -126,7 +126,7 @@ class UploadPipelineDialog extends React.Component<UploadPipelineDialogProps, Up
                     endAdornment: (
                       <InputAdornment position='end'>
                         <Button color='secondary' onClick={() => this._dropzoneRef.current!.open()}
-                          style={{ padding: '3px 5px', margin: 0 }}>
+                          style={{ padding: '3px 5px', margin: 0, minWidth: 90 }}>
```

Comment:
How about `white-space: nowrap;`? This way you don't need to calculate the width based on the text and font.

Reply:
Considered this. Done